python/sqlflow_submitter/pai/utils.py

```diff
@@ -70,5 +70,15 @@ def oss_temporary(ak, sk, endpoint, filename):
                               name)  # This would never happen.
     else:
         bucket.put_object_from_file(name, filename)
-    yield f'oss://{bucket_name}.{internal_endpoints[endpoint]}/{name}'
+    yield f'oss://{bucket_name}.{oss_internal_endpoints[endpoint]}/{name}'
     bucket.delete_object(name)
+
+
+def copyfileobj(source, dest, ak, sk, endpoint, bucket_name):
+    '''
+    copy_file_to_oss copies (`source`(local file) to an object on OSS
+    '''
+    auth = oss2.Auth(ak, sk)
```

Comment:
If we can save the explain result picture in the bucket that saves the model, we can use Tensorflow gfile, so that we don't need to configure AK/SK again.

Reply:
Good idea. I will take a try later. issue #1872